stable/dask/templates/dask-jupyter-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "dask.jupyter-fullname" . }}-config
+  labels:
+    app: {{ template "dask.name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
Once you've updated the `_helpers.tpl`, use `chart: {{ template "dask.chart" . }}`.